action-ssh-to-runner/action.yaml

```diff
@@ -17,27 +17,70 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: "SSH to runner: Package dependencies for tmate"
+    - name: "SSH to runner: Write tmate config file"
       shell: bash
       run: |
-        ${{ inputs.sudo }} && sudo_cmd="sudo" || sudo_cmd=""
-        # mxschmitt/action-tmate does not have a check to use yum/dnf instead of apt.
-        # For now, all dependencies needed by tmate are already installed.
-        test -f /etc/lsb-release || $sudo_cmd ln -s /bin/true /bin/apt-get
-    - name: "SSH to runner: Print usage information"
+        TMATE_CONF=/tmp/tmate.conf
+        echo "TMATE_CONF=$TMATE_CONF" >> $GITHUB_ENV
+        echo "set -g tmate-server-host ${{ inputs.tmate-server-host }}" >> ${TMATE_CONF}
+        echo "set -g tmate-server-port ${{ inputs.tmate-server-port }}" >> ${TMATE_CONF}
+        echo "set -g tmate-server-rsa-fingerprint ${{ inputs.tmate-server-rsa-fingerprint }}" >> ${TMATE_CONF}
+        echo "set -g tmate-server-ed25519-fingerprint ${{ inputs.tmate-server-ed25519-fingerprint }}" >> ${TMATE_CONF}
+        cat ${TMATE_CONF}
```

Comment:
It would be best (but not mandatory) to set those config in a file here, and use `envsubst` to modify it appropriately.

Note: inputs var are set in the env as: `INPUT_UPPERCASE-INPUT-VAR`, example for `tmate-server-host` it would be `INPUT_TMATE-SERVER-HOST`

Reply:
Argh, this inconsistent behavior is a pain. Good to know, thanks!